app/views/images/index.html.haml

```diff
@@ -0,0 +1,45 @@
+.container-fluid.col-sm-7.col-sm-offset-2
+  - if @images.any?
+    - @images.reverse.each do |image|
+      .instagram-container.container-fluid.col-sm-12
+        .container-fluid.col-sm-12#username
+          .container-fluid.col-sm-8
+            = image.user.email
+          .container-fluid.col-sm-4.text-right
+            = relative_time(image.created_at)
+        #imagecontainer.container-fluid.col-sm-12
```

Comment:
Classes should be listed before IDs (.container-fluid should precede #imagecontainer)